Patch to modify build system to follow autoreconf suggestions and to force the
update of everything.
==========================================================================
--- a/Makefile.am
+++ b/Makefile.am
@@ -1,6 +1,8 @@
 # $Id: Makefile.am,v 1.22 2008/10/01 05:33:56 asuraparaju Exp $
 #
 
+ACLOCAL_AMFLAGS = -I m4
+
 SUBDIRS = libdirac_byteio libdirac_common libdirac_motionest libdirac_encoder \
           libdirac_decoder encoder decoder util $(CPPUNITTESTS_DIR) \
           tests $(DOC_DIR) win32
--- a/bootstrap
+++ b/bootstrap
@@ -10,12 +10,12 @@
 rm -rf autom4te.cache
 
 set -x
-aclocal -I m4
+aclocal --force -I m4
 libtoolize --force --copy
-automake --foreign --copy --add-missing
+automake --foreign --copy --add-missing --force-missing
 if [ $? -ne 0 ]; 
 then
 	echo "automake failed" 
 	exit -3 
 fi
-autoconf
+autoconf --force
--- a/configure.ac
+++ b/configure.ac
@@ -57,9 +57,7 @@
 AM_PROG_LIBTOOL
 AC_SYS_LARGEFILE
 
-ACLOCAL_AMFLAGS="-I m4 $ACLOCAL_AMFLAGS"
-AC_SUBST(ACLOCAL_AMFLAGS)
-
+AC_CONFIG_MACRO_DIR([m4])
 
 dnl -------------------------------------------------------------------------
 dnl Check if compiling using MS VC++ compiler in msys environment on Windows
